### PR TITLE
[15.0][ADD] product_attribute_description

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

File: product_attribute_description/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "author": "ERP Harbor Consulting Services,"
+    "Nitrokey GmbH,"
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "summary": """
+    Product Attribute Description
+    This module Product Attribute Description is a part of product and used to show
+    attribute description of the product.
+     """,
+    "depends": ["website_product_attribute_description", "product"],
```
Comment (rousseldenis) [inline product_attribute_description/__manifest__.py]: You have recursive dependency here:

```suggestion
    "depends": ["product"],
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: product_attribute_description/__manifest__.py (hunk 1 above)
Comment (dsolanki-initos) [reply product_attribute_description/__manifest__.py]: Okay, I'll fix the recursive dependency.

**Comment by dsolanki-initos** [comment]
@rousseldenis I've done the changes as per your feedback, Please review it.

File: product_attribute_description/README.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to "Product -> Select Attribute -> Add your description"
+#. To check you description Go to the Shop(website) find relevant product of attribute
```
Comment (rousseldenis) [inline product_attribute_description/README.rst]: You should add something that says people have to install website related module.

**Review by rousseldenis — APPROVED** [review]
Minor comment

File: product_attribute_description/README.rst (hunk 2 above)
Comment (dsolanki-initos) [reply product_attribute_description/README.rst]: Sure, I'll update it.
Thank you

Comment (dsolanki-initos) [reply product_attribute_description/README.rst]: @rousseldenis, I've updated the README.rst

**Review by rousseldenis — APPROVED** [review]
Could you squash your commits ?

**Pull request closed without merging**